aiida/manage/configuration/options.py

```diff
@@ -185,6 +185,22 @@
         'description': 'Boolean whether to print AiiDA deprecation warnings',
         'global_only': False,
     },
+    'transport.task_retry_initial_interval': {
+        'key': 'task_retry_initial_interval',
+        'valid_type': 'int',
+        'valid_values': None,
```

Comment:
would be good here if we could validate for `>0`

Reply:
I could imagine changing this `valid_type`/`valid_values` into something akin to a [jsonschema](https://pypi.org/project/jsonschema/) or just allowing it to be callable.

e.g.

```json
{
  `validator`: {'type': 'integer', 'minimum': 1}
}
```

or

```json
{
  `validator`: lambda: v: isinstance(v, int) and v > 0
}
```

Reply:
Extending the validation is definitely a good idea, but perhaps we should open a new issue/PR for this? Note that the `valid_type` is also used to parse the option value, so I'm not sure if we can just merge `valid_type`/`valid_values` into one `validator` function.